website_sale_hide_price/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Tecnativa - David Vidal
+# Copyright 2018 Trey, kilobytes de soluciones - Abraham Gonzalez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    'name': 'Website Sale Hide Price',
+    'version': '10.0.1.0.0',
+    'category': 'Website',
+    'author': 'Trey, Kilobytes de soluciones, '
```

Comment:
You shouldn't add your company here unless you make significant changes on the module, which is not the case.